template/v3/Dockerfile

```diff
@@ -15,12 +15,26 @@ ARG NB_GID=100
 
 # https://www.openssl.org/source/
 ARG FIPS_VALIDATED_SSL=3.0.8
+ARG MIN_REQUIRED_MICROMAMBA_VERSION=1.5.11
```

Comment:
For V3 image, we might want to update base image instead of pinning the version. Let me check with the team internally and get back to you

Reply:
ack, will not change v3 Dockerfile and track issue separately in https://github.com/aws/sagemaker-distribution/issues/535